.github/CONTRIBUTING.md

```diff
@@ -1,35 +1,50 @@
-# Web Server
+# Contributing
+
+## Web Server
+
+### Vagrant
+
+For simplicity a Vagrant file is included. To start you will need:
+
+* [Vagrant](http://www.vagrantup.com/downloads.html)
+* [VirtualBox](https://www.virtualbox.org/wiki/Linux_Downloads)
 
-## PHP Web Server
+Then simply run `vagrant up`.
 
-For a simple local development environment, you will need:
+### Simple PHP Server
 
-* [A supported version of PHP](http://php.net/supported-versions.php)
-* [Node.js](https://nodejs.org/)
-* `php7.0-curl`
-* `php7.0-intl`
-* `php7.0-json`
-* `php7.0-mbstring`
-* `php7.0-xml`
-* packages installed with `npm install`
+Alternatively, for a simple local development environment running on PHP, you will need:
+
+* [A supported version of PHP](http://php.net/supported-versions.php) with:
+  * `php7.0-cli`
+  * `php7.0-fpm`
+  * `php7.0-curl`
```

Comment:
Is `php7.0-fpm` required if you just run cli? I thought that was only required with nginx.